authoring/tools-for-developers.md

```diff
@@ -0,0 +1,80 @@
+# Tools for developers
+
+This section presents some useful tools that can be very helpful for the development workflow.
+
+
+## Code linters
+
+Code linteres are tools that analyze source code and identify programming errors, stylistic errors,
+invalid typing, wrong or missing docstrings, etc.
+
+Some linters that could help you to keep your code in good shape are:
+
+- [flake8](https://flake8.pycqa.org/en/latest/)
```

Comment:
I think in general it's best to just recommend one, and then we can have a list of others afterward. That way a reader that just "wants to be told what to do" has a path forward, and one that wants to do their research has options to look into.